bin/ares-config.js

```diff
@@ -99,20 +100,20 @@ function config() {
     log.verbose("profile()", "options:", options);
 
     if (!Object.prototype.hasOwnProperty.call(configFiles, options.profile)) {
-      return finish(errHndl.changeErrMsg("INVALID_VALUE", "profile", options.profile));
```

Comment:
If finish is in errhndl as it is now, it would be better to call errHndl.changeErrMsg in finish function.

Reply:
I moved finish() of errHandle to each bin level command.js file.